server/routes/createGoal/createGoalController.ts

```diff
@@ -2,16 +2,26 @@ import type { RequestHandler } from 'express'
 import type { Prisoner } from 'prisonRegisterApiClient'
 import type { PrisonerSummary } from 'viewModels'
 import createError from 'http-errors'
+import EducationAndWorkPlanService from '../../services/educationAndWorkPlanService'
 import PrisonerSearchService from '../../services/prisonerSearchService'
 import CreateGoalView from './createGoalView'
 import AddStepView from './addStepView'
 import AddNoteView from './addNoteView'
 import validateCreateGoalForm from './createGoalFormValidator'
 import parseDate from '../parseDate'
 import validateAddStepForm from './addStepFormValidator'
+import CreateGoalFormToCreateGoalDtoMapper from './createGoalFormToCreateGoalDtoMapper'
+import logger from '../../../logger'
 
 export default class CreateGoalController {
-  constructor(private readonly prisonerSearchService: PrisonerSearchService) {}
+  constructor(
+    private readonly prisonerSearchService: PrisonerSearchService,
+    private readonly educationAndWorkPlanService: EducationAndWorkPlanService,
+  ) {}
+
+  private static createDtoMapper(): CreateGoalFormToCreateGoalDtoMapper {
```

Comment:
Same comment as above re: defining your mapper as a simple exported function rather than a class, then you dont need this static etc